packages/blue-sdk/src/market/MarketUtils.ts

```diff
@@ -137,26 +137,22 @@ export namespace MarketUtils {
    * @param utilization The target utilization rate (scaled by WAD).
    */
   export function getSupplyToUtilization(
-    {
-      totalSupplyAssets,
-      totalBorrowAssets,
-    }: {
+    market: {
       totalSupplyAssets: BigIntish;
       totalBorrowAssets: BigIntish;
     },
     utilization: BigIntish,
   ) {
     utilization = BigInt(utilization);
-    totalBorrowAssets = BigInt(totalBorrowAssets);
     if (utilization === 0n) {
-      if (totalBorrowAssets === 0n) return totalSupplyAssets;
+      if (getUtilization(market) === 0n) return 0n;
```

Comment:
This case depends a lot on the business case no?

Reply:
Yes, so I updated jsdocs so it's clearer thanks